reference-sheets/Simulation.md

```diff
@@ -0,0 +1,34 @@
+# Robot Simulation
+
+Simulation in WPILib allows for code and logic to be tested onboard your computer, rather than physical hardware. It's exceptionally useful for when construction and electronics have not finished the robot, but you have code to run!
```

Comment:
```suggestion
Simulation in WPILib allows for code and logic to be tested onboard your computer, rather than with physical hardware. It's exceptionally useful for when mechanical hasn't finished the robot, but you have code to run!
```